### PR TITLE
chore(client-app): merge modal package into client-app

### PR DESCRIPTION
**Pull request by amritk**
- merged the client-modal into the client app
- moved the adding of the scalar container classes to a helper in components 
- and its not best practice, but duplicated some helpers from oas-utils into components to remove components dependency on oas. now it should no longer build any time we change an entitity speeding up build times by a lot!

### CODE REVIEW

File: packages/client-app/src/views/Request/RequestSection/RequestAuth.vue [hunk 1]
```diff
@@ -175,7 +175,7 @@ const updateScheme: UpdateScheme = (path, value) =>
               id="http-basic-username"
               class="text-c-2"
               :modelValue="activeSecurityScheme.scheme.value"
-              placeholder="Username"
+              placeholder="ScalarEnjoyer01"
```
Comment (antlio) [inline packages/client-app/src/views/Request/RequestSection/RequestAuth.vue:178]: 😍

File: packages/client-app/src/views/Request/RequestSection/RequestAuth.vue [hunk 2]
```diff
@@ -184,7 +184,7 @@ const updateScheme: UpdateScheme = (path, value) =>
             <RequestAuthDataTableInput
               id="http-basic-password"
               :modelValue="activeSecurityScheme.scheme.secondValue"
-              placeholder="Password"
+              placeholder="xxxxxx"
```
Comment (antlio) [inline packages/client-app/src/views/Request/RequestSection/RequestAuth.vue:187]: maybe?
```suggestion
              placeholder="••••••"
```

Comment (amritk) [reply packages/client-app/src/views/Request/RequestSection/RequestAuth.vue:187]: Ya I tried this but is it too close to an actual value?

Comment (antlio) [reply packages/client-app/src/views/Request/RequestSection/RequestAuth.vue:187]: yes not wrong, color is dimmed down, maybe we should stick to password?

Comment (amritk) [reply packages/client-app/src/views/Request/RequestSection/RequestAuth.vue:187]: My issue with password is it's the same as the label so doesn't really help. The other thing we could do is a fake password like 123ABC

**Review by marclave — APPROVED** [review]
alright lets get this in.

we can FF with:
- remove duplicate helpers amrit mentioned
- fix react
- fix docusuarus

**Pull request merged**